cognite/client/_api/functions.py

```diff
@@ -192,11 +192,10 @@ def create(
         sleep_time = 1.0  # seconds
         for i in range(MAX_RETRIES):
             file = self._cognite_client.files.retrieve(id=file_id)
-            if file is None or not file.uploaded:
-                time.sleep(sleep_time)
-                sleep_time *= 2
-            else:
+            if file is not None and file.uploaded:
```

Comment:
Can be further simplified to
```suggestion
            if file and file.uploaded:
```

Reply:
Nice one, thanks Erlend!